File: NxNGOL/src/cell.py
class cell:
 
    #constructor method for the obejct 
    def __init__(self, alive=False):
        self.alive = alive

    #method which returns the cell's alive parameter --> Is this nec. if not private variables?
    def returnCellStatus(self):
        return self.alive

    #method which prints a given cell's __alive parameter
    def printCellStatus(self):
        if self.alive:
            print("A", end='')
        else:
            print("D", end='')

    #method which flips a cell's alive parameter (i.e. A->D, D->A)
    def flipStatus(self):
        self.alive = not (self.alive)

File: NxNGOL/srcJava/cell.java
public class cell{
	
	///// WE DEFINE OUR NON-CONSTANT PARAMETERS /////
	private boolean alive;
	
	// constructor method
	public cell (boolean alive){
		// we initialize all our cells and determine its neighbors
		this.alive = alive;
	}// end of constructor 
	
	// method which returns the cell's alive parameter
	public boolean returnCellStatus(){
		return this.alive;
	}// end of returnCellStatus() method
	
	// method which prints a given cell's alive parameter, (A=alive & B=dead)
	public void printCellStatus(){
		if (alive){
			System.out.print("A");
		}
		else{
			System.out.print("D");
		}
	}// end of printCellStatus() method
	
	// method which flips a cell's alive status (i.e. A->D, D->A)
	public void flipStatus(){
		// we flip the status of the cell 
		alive = !alive;
	}// end of flipStatus() method 
	
}// end of class
